tools/PI/DevHome.PI/Controls/ExpandedViewControl.xaml

```diff
@@ -30,22 +30,32 @@
                     <RowDefinition Height="Auto"/>
                 </Grid.RowDefinitions>
 
-                <ListView
-                x:Name="NavLinksList" SelectionMode="Single" VerticalAlignment="Stretch" ItemsSource="{x:Bind viewModel.Links}"
-                 SelectedIndex="{x:Bind viewModel.SelectedNavLinkIndex, Mode=TwoWay}" SelectionChanged="{x:Bind viewModel.Navigate}">
-                    <ListView.ItemTemplate>
-                        <DataTemplate>
-                            <StackPanel Orientation="Horizontal">
-                                <TextBlock Text="{Binding IconText}" Style="{x:Null}"
-                                FontFamily="Segoe Fluent Icons" FontSize="12"
-                                HorizontalAlignment="Left" VerticalAlignment="Center"/>
-                                <TextBlock Text="{Binding ContentText}"
-                                Margin="12,0,0,0" HorizontalAlignment="Left" VerticalAlignment="Center"/>
-                            </StackPanel>
-                        </DataTemplate>
-                    </ListView.ItemTemplate>
-                </ListView>
-
+                <StackPanel Orientation="Vertical">
+                    <ListView
+                    x:Name="NavLinksList" SelectionMode="Single" VerticalAlignment="Stretch" ItemsSource="{x:Bind viewModel.Links}"
+                     SelectedIndex="{x:Bind viewModel.SelectedNavLinkIndex, Mode=TwoWay}" SelectionChanged="{x:Bind viewModel.Navigate}">
+                        <ListView.ItemTemplate>
+                            <DataTemplate>
+                                <StackPanel Orientation="Horizontal">
+                                    <TextBlock Text="{Binding IconText}" Style="{x:Null}"
+                                    FontFamily="Segoe Fluent Icons" FontSize="12"
+                                    HorizontalAlignment="Left" VerticalAlignment="Center"/>
+                                    <TextBlock Text="{Binding ContentText}"
+                                    Margin="12,0,0,0" HorizontalAlignment="Left" VerticalAlignment="Center"/>
+                                </StackPanel>
+                            </DataTemplate>
+                        </ListView.ItemTemplate>
+                    </ListView>
+                    <Rectangle x:Name="Separator"
+                        Visibility="{x:Bind viewModel.AppSettingsVisibility, Mode=OneWay}"
+                        RadiusX="{ThemeResource AppBarSeparatorCornerRadius}"
+                        RadiusY="{ThemeResource AppBarSeparatorCornerRadius}"
+                        Height="{ThemeResource AppBarSeparatorWidth}"
+                        Width="200"

```

Comment:
Can `Width` be bound to the width of another UI element?  Maybe the ListView?

Reply:
I don't see a way to do this in XAML, just in the code behind. Are you aware of a way to do it?

Reply:
`{x:Bind [ElementName].Width} I believe.  XAML allows referencing other UI elements by name.

Reply:
Another option is `Binding` Banner.xaml uses binding to to the height of a parent element.

Reply:
The Bind seemed to work. Thanks!